### ISSUE TITLE
Unable to download dataset via DropBox

### ISSUE THREAD

**Issue by noor-fathima-khanum** [RyanXingQL/MFQEv2.0#12]
Hello, I am trying to download the dataset mentioned in this paper. However, the dropbox link is broken.

**Comment by noor-fathima-khanum** [comment]
Hello, Thank you for your quick response and fix! This is very helpful!

**Comment by ryanxingql** [comment]
Hi @claudiom4sir ,
as mentioned in [here](https://github.com/ryanxingql/mfqev2.0/wiki/MFQEv2-Dataset), our data-set can be downloaded at [this link](https://www.dropbox.com/sh/tphdy1lmlpz7zq3/AABR4Qim-P-3xGtouWk6ohi5a?dl=0).

Can you download it now?

And, we don't need `test_18.z02`. Try these commands for unzip:

```bash
zip -s 0 test.z* --out unsplit.zip
unzip unsplit.zip -d test/
```

**Comment by claudiom4sir** [comment]
Hi @ryanxingql,
the link you posted is the one on the homepage and it works, I had already downloaded all the files. My problem was during decompression.

After some trials I found the problem. It seems that some files I downloaded from the browser were somehow corrupted. It is very strange but I fixed it by downloading these files again, until the unzip processed worked.

On Ubuntu I ran the command `zip -s 0 train_108 -d train.zip` followed by `unzip train -d .` for train_108 and the same for test_18 and it worked.

Problem solved, thank you!
